calyx-py/calyx/builder_util.py

```diff
@@ -0,0 +1,175 @@
+# pylint: disable=import-error
+import calyx.builder as cb
+
+
+def insert_eq(comp: cb.ComponentBuilder, left, right, cellname, width):
```

Comment:
Note for future work, beyond this PR: it would be cool if the builder had a central "gensym" utility for generating fresh names. Then these utilities wouldn't need to take in a group and cell name (or at least they could be optional); the builder would just call them `lt1234` or whatever automatically.

Reply:
Now its own issue: #1635! Resolving this conversation in the meanwhile